src/sgame/components/SpawnerComponent.cpp

```diff
@@ -40,6 +40,7 @@ void SpawnerComponent::Think(int timeDelta) {
 	if (blocker) {
 		if (!blocker->oldEnt) {
 			logger.Warn("Spawn blocking entity has oldEnt == nullptr");
```

Comment:
When does this happen?

Reply:
I don't think so. The CBSE pass loose that information, and it might be many things, depending on what the code becomes.